jdi-light-angular/pom.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jdi-light-angular</artifactId>
+    <name>JDI Angular</name>
+    <dependencies>
+        <dependency>
+            <groupId>com.epam.jdi</groupId>
+            <artifactId>jdi-light</artifactId>
+            <version>1.2.14</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.qameta.allure</groupId>
+            <artifactId>allure-testng</artifactId>
+            <version>${allure.testng}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.epam.jdi</groupId>
+            <artifactId>jdi-light-html</artifactId>
+            <version>1.2.14</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>3.3.3</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <groupId>com.epam.jdi</groupId>
+
+    <properties>
+        <aspectj.version>1.9.1</aspectj.version>
+<!--        <aspectj.maven>1.11</aspectj.maven>-->
+        <java.version>1.8</java.version>
+        <allure.testng>2.13.1</allure.testng>
+        <allure.maven>2.10.0</allure.maven>
+        <jetty.version>9.4.12.RC2</jetty.version>
+    </properties>
+
+    <version>1.2.14</version>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>${java.version}</source>
+                    <target>${java.version}</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <!--                <groupId>org.codehaus.mojo</groupId>-->
+                <!--                <artifactId>aspectj-maven-plugin</artifactId>-->
+                <!--                <version>${aspectj.maven}</version>-->
+                <groupId>com.nickwongdev</groupId>
```

Comment:
do we really need it here? Please remove

Reply:
Thanks, removed